content/about/_index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: About
+summary: Pulumi is a Seattle-based company that enables teams to create, deploy, and manage modern cloud applications and infrastructure.
```

Comment:
We should use `meta_desc` for these instead of `summary` and remove our use of `summary` in the small number of places it's used. `summary` is only used in a handful of blog posts and was a holdover from the initial blog migration when Chris was originally trying to collapse the summary and meta description into a single thing -- but we've since gone back to allowing separate meta descriptions and summaries (and the summaries are now obtained via `.Summary`, Hugo's [Content Summaries](https://gohugo.io/content-management/summaries/)).